src/Storages/Streaming/StorageRandom.cpp

```diff
@@ -445,7 +456,7 @@ class GenerateRandomSource final : public ISource
     Chunk doGenerate(UInt64 block_size_)
     {
         if (block_size_ == 0)
-            return header_chunk.clone();
+            return {};
```

Comment:
In prev implementation, the `block_size = 0`, means propagate an empty heart chunk.
I think it's broken changes

Reply:
can you try streaming query again ?